includes/class-newspack-popups-model.php

```diff
@@ -1092,7 +1101,7 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 					<div class="newspack-popup__content">
 						<?php echo do_shortcode( $body ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 					</div>
-					<button class="newspack-lightbox__close" style="color: <?php echo esc_attr( $close_button_color ); ?>"aria-label="<?php esc_html_e( 'Close Pop-up', 'newspack-popups' ); // phpcs:ignore WordPressVIPMinimum.Security.ProperEscapingFunction.htmlAttrNotByEscHTML ?>">
+					<button class="newspack-lightbox__close" style="background-color: <?php echo esc_attr( $close_button_background_color ); ?>; color: <?php echo esc_attr( $close_button_color ); ?>"aria-label="<?php esc_html_e( 'Close Pop-up', 'newspack-popups' ); // phpcs:ignore WordPressVIPMinimum.Security.ProperEscapingFunction.htmlAttrNotByEscHTML ?>">
```

Comment:
Nit: is there any reason to render `style="background-color: "` if `$close_button_background_color` is empty?

Reply:
Absolutely not! 😄 I reworked this a bit in [9e7f727](https://github.com/Automattic/newspack-popups/pull/1396/commits/9e7f72725178c40781eb789c6201a18dcb295bb2) so it only writes the `background-color` if it's needed.